Detectors/TOF/workflow/include/TOFWorkflowUtils/TOFDigitWriterSpitterSpec.h

```diff
@@ -0,0 +1,160 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef O2_TOFDIGIT_SPLITTER_WRITER_H
+#define O2_TOFDIGIT_SPLITTER_WRITER_H
+
+/// @file   TOFDigitWriterSplitterSpec.h
```

Comment:
The file name is actually ..Spitter.. rather than ..Splitter... Typo ?

Reply:
Ah... it was a typo. Fixed now. Thanks